README.md

```diff
@@ -500,6 +500,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
   * [cargo-multi](https://crates.io/crates/cargo-multi) — runs specified cargo command on multiple crates [![build badge](https://api.travis-ci.org/imp/cargo-multi.svg?branch=master)](https://travis-ci.org/imp/cargo-multi)
   * [cargo-outdated](https://crates.io/crates/cargo-outdated) — displays when newer versions of Rust dependencies are available, or out of date [![build badge](https://api.travis-ci.org/kbknapp/cargo-outdated.svg?branch=master)](https://travis-ci.org/kbknapp/cargo-outdated)
   * [cargo-release](https://crates.io/crates/cargo-release) — tool for releasing git-managed cargo project, build, tag, publish, doc and push [![Rust](https://github.com/crate-ci/cargo-release/actions/workflows/ci.yml/badge.svg)](https://github.com/crate-ci/cargo-release/actions/workflows/rust.yml)
+  * [cargo-rpm](https://github.com/iqlusioninc/cargo-rpm) [![CRATE](https://crates.io/crates/cargo-rpm)] — cargo subcommand for building .rpm releases of Rust projects.
```

Comment:
That second link looks like you were trying for an image, but hasn't quite worked?

Reply:
I'd taken the template from the `CONTRIBUTING.md` file:

> if you want to add something, please use the template `[ACCOUNT/REPO](https://github.com/ACCOUNT/REPO) [![CRATE](https://crates.io/crates/CRATE)] — DESCRIPTION`

Although having copied that now, I presume `CRATE` at the very least should be `cargo-rpm`…? Or have I misinterpreted the above?

Reply:
I've updated the `CRATE` reference to be correct but also dropped the `!` in the crates.io link—I've updated the `CONTRIBUTING.md` to remove this as it doesn't appear to be correct (as you rightly pointed out, it wants to interpret that as an image.)

Reply:
Thanks, yup. Our formatting needs a bit of work at some point, but thanks for fixing that bit.

Reply:
Thanks!